pkg/pgmodel/migrate.go

```diff
@@ -223,9 +239,19 @@ func ensureVersionTable(db *pgxpool.Pool) error {
 	return nil
 }
 
-func getDBVersion(db *pgxpool.Pool) (semver.Version, error) {
+func getSchemaVersion(db *pgxpool.Pool) (semver.Version, error) {
+	var version semver.Version
```

Comment:
Sorry, but I didn't get where are we initializing this.

Reply:
Looks like this is just a placeholder value to send when an error occurs that should be ignored.

Reply:
Yeah, it's pretty silly. In my defense, it was like that when I found it 😉